launch/client.py

```diff
@@ -2759,6 +2765,77 @@ def create_llm_model_endpoint(
             else:
                 raise ValueError("Endpoint should be one of the types 'sync' or 'async'")
 
+    def list_llm_model_endpoints(self) -> List[Endpoint]:
```

Comment:
Discussing this in Slack, but I wonder if we can take this opportunity to axe the handcrafted classes like `Endpoint`.

Reply:
Alternatively, we can use pydantic models for the response models.